root/usr/share/nethesis/NethServer/Module/Dashboard/SystemStatus/Backup.php

```diff
@@ -48,7 +48,45 @@ private function readBackup()
         $backup['vfs'] = $br['VFSType'] ? $br['VFSType'] : '-';
         $backup['status'] = $br['status'];
         $backup['time'] = $br['BackupTime'];
-
+        
+        $orario = explode(" ",$backup['time']);
+      
+        function formatOre($h){
+	        if(strlen($h)==1){return "0".$h;}else{return $h;}
+        }
+        
+        function formatMin($m){
+	         if(strlen($m)==1){return "0".$m;}else{return $m;}
+        }
+        
+        function formatDay($d){
```

Comment:
You could covert the entire function in a couple of line, something like:
```
$days = array("Sunday", "Monday" ...)
return $days[$d];
```

Also bear in mind that even the days must be translated!